commons-numbers-complex/src/main/java/org/apache/commons/numbers/complex/Complex.java

```diff
@@ -1583,7 +1299,7 @@ public Complex sin() {
         // Define in terms of sinh
```

Comment:
This comment is not relevant any more. However it is missing from ComplexFunctions so you should move it there.